pages/form-builder/responses/[[...params]].tsx

```diff
@@ -55,12 +55,19 @@ const Responses: NextPageWithLayout<ResponsesProps> = ({
   const navItemClasses =
     "no-underline !shadow-none border-black border-1 rounded-[100px] pt-1 pb-2 laptop:py-2 px-5 mr-3 mb-0 text-black visited:text-black focus:bg-[#475569] hover:bg-[#475569] hover:!text-white focus:!text-white [&_svg]:focus:fill-white";
 
+  if (!isAuthenticated) {
+    return (
+      <>
+        <PageTemplate title={t("responses.title")}>
+          <LoggedOutTab tabName={LoggedOutTabName.RESPONSES} />
+        </PageTemplate>
+      </>
+    );
+  }
+
   if (deliveryOption && deliveryOption.emailAddress) {
     return (
       <>
-        <Head>
```

Comment:
Removed `Head` as it is already included in the `PageTemplate` component we use below.